documentation/book/oauth/proc-oauth-server-config.adoc

```diff
@@ -0,0 +1,58 @@
+// Module included in the following assemblies:
+//
+// assembly-oauth-strimzi-config.adoc
+
+[id='proc-oauth-server-config-{context}']
+= Deploying an {oauth} authorization server
+
+This procedure describes how to deploy an authorization server and configure it for integration with {ProductName}.
+
+Depending on the chosen xref:con-oauth-authentication-client-options-{context}[authentication flow], you can use the authorization server to configure authentication credentials for:
+
+* Configure brokers
+* Configure and authorize clients
+* Configure users
+* Obtain access and refresh tokens
+
+User credentials are managed by an authorization server using a _realm_, which describes security policies.
+The realm is the authentication provider for clients.
+
+To use {oauth} with {ProductName}, you need a deployment of an authorization server to be able to create and manage authentication realms.
+
+NOTE: If you already have an authorization server deployed, you can skip the deployment step and use your current deployment.
+
+.Prerequisites
+
+* {ProductName} and Kafka is running
+
+For the authorization server deployment:
+
+* Check the supported configuration for the authorization server
+* Installation requires a user with cluster-admin role, such as system:admin
+
+.Procedure
+
+. Deploy the authorization server to your cluster.
+. Access the CLI or admin console for the authorization server to create the {oauth} policies for {ProductName}.
++
+Consult the product documentation for the authorization server for information on creating the policies.
+
+. Create a _realm_.
++
+* Enable the realm
+* Set SSL as `external`
+* Set session and token timeouts
+
+. Create the following _clients_ and configure their authentication credentials:
++
+--
+* Kafka
```

Comment:
Could we clarify what are these needed for? 
* I think we should make this more generic. 
* The user will probably need a lot more than just one producer and one consumer user. 
* Also a user for Connect, mirror maker or bridge etc. might be needed. 
* What is the difference between the KAfka and Kafka broker clients?

I would really jsut make this generic. `Create the _clients_ and configure their authentication credentials.` and remove the list.

Reply:
Made more generic since the review by @mstruk